src/pages/journal/journal.ts

```diff
@@ -74,7 +81,7 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
       isLoading$.pipe(map(this.handleLoadingSpinner))
     );
     this.createLoadingSpinner();
-    this.subscription = merged$.subscribe();
+    this.subscription = merged$.subscribe((data) => this.createSlots(data));
```

Comment:
could it just be `.subscribe(this.createSlots)`?

Reply:
done :)